Form-Controls/index.html

```diff
@@ -33,18 +33,19 @@ <h2>Collecting Data</h2>
 <fieldset>
   <legend>Choose one of our colour:</legend>
 
- <p> <input type="radio" name="colour" id="color_1" value="Red"> 
+ <p> <input type="radio" name="colour" id="color_1" value="Red" required> 
   <label for="color_1">Small</label> </p>
-  <p> <input type="radio" name="colour" id="color_2" value="Green"> 
+  <p> <input type="radio" name="colour" id="color_2" value="Green" required> 
     <label for="color_2">Medium</label> </p>
-    <p> <input type="radio" name="colour" id="color_3" value="Black"> 
+    <p> <input type="radio" name="colour" id="color_3" value="Black" required> 
       <label for="color_3">Large</label> </p>
         </fieldset>
      <br>
   
   <p>
```

Comment:
Great work,  There are also many other types we can use in input element, you can them out;
https://www.w3schools.com/html/html_form_input_types.asp